app/views/tasks/_no_tasks.html.haml

```diff
@@ -2,4 +2,4 @@
   .ccs-task__information
     .ccs-task__body
       %p All your tasks are complete.
-      %p You’re up to date on all frameworks.
+      %p You’re up to date on all frameworks.
```

Comment:
I've seen this happen before - did I hear a rumour you had switched to Rubymine™?